app/jobs/regular/process_post.rb

```diff
@@ -32,7 +32,7 @@ def execute(args)
           # TODO suicide if needed, let's gather a few here first
           Rails.logger.warn("Cooked post processor in FATAL state, bypassing. You need to urgently restart sidekiq\norig: #{orig_cooked}\nrecooked: #{recooked}\ncooked: #{cooked}\npost id: #{post.id}")
         else
-          post.update_column(:cooked, cp.html)
```

Comment:
`update_column` skips callbacks so I have a hunch that rebaking doesn't acutally update the index.